src/service.ts

```diff
@@ -0,0 +1,11 @@
+import users from './api/users';
+import { ToDo } from './types/todo';
+import { User } from './types/user';
+
+export function findUserById(id: number): User {
+  return users.filter(user => user.id === id)[0];
+}
+
+export function getTodosMaxId(todosList: ToDo[]): number {
+  return Math.max(...todosList.map(todo => todo.id));
```

Comment:
The `getTodosMaxId` function assumes the `todosList` is not empty. Consider adding a check to handle empty arrays, such as returning 0 or another default value if `todosList` is empty.